Include/typeslots.h

```diff
@@ -88,3 +88,51 @@
 /* New in 3.5 */
 #define Py_tp_finalize 80
 #endif
+#define Py_tp_name 81
+#define Py_tp_basicsize 82
+#define Py_tp_itemsize 83
+#define Py_tp_vectorcall_offset 84
+#define Py_tp_flags 85
+#define Py_tp_weaklistoffset 86
+#define Py_tp_dict 87
+#define Py_tp_dictoffset 88
+#define Py_tp_mro 89
+#define Py_tp_cache 90
+#define Py_tp_subclasses 91
+#define Py_tp_weaklist 92
```

Comment:
Please don't add slots for data (as opposed to functions).

Reply:
make sense. I will update it tomorrow.